quattor/functions/validation.pan

```diff
@@ -72,27 +72,27 @@ function is_profile_list = {
     profiles = ARGV[0];
     errors_ni = '';
     errors_na = '';
-    ok = first(profiles,k,v);
+    ok = first(profiles, k, v);
```

Comment:
actuaully, what does this horrible code try to do? can it be replaced by a format?